etc/profile-m-z/nicotine.profile

```diff
@@ -58,6 +58,8 @@ private-dev
 private-tmp
 
 dbus-user none
+dbus-user.own org.nicotine_plus.Nicotine
+dbus-user.talk ca.desrt.dconf
```

Comment:
Add `dbus-user.talk org.freedesktop.portal.Fcitx` below this line for the Fcitx fix.

Reply:
I haven't thought of adding the Fcitx part. Isn't it something that should be optional? At least I don't see it being in other profiles.

Reply:
We do have https://github.com/netblue30/firejail/blob/2a8621cd940262397b048881d7fc34e5ba9644aa/etc/profile-a-l/linuxqq.profile#L32

But you're correct. Let's keep it optional :+1: . I'll fix linuxqq later. Thanks for the PR!